lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -365,6 +367,18 @@ def add_address(post, options)
         end
       end
 
+      def add_statement_address(post, options)
+        return unless statement_address = options[:statement_address]
+        return unless [:address1, :city, :zip, :state].all? { |key| statement_address[key].present? } 
+
+        post[:statement_address] = {}
+        post[:statement_address][:line1] = statement_address[:address1]
+        post[:statement_address][:line2] = statement_address[:address2]
```

Comment:
oh yeah i meant to have a check on that line to make sure address2 was present. The reason i didnt check above is that its not required.